fedmsg_meta_fedora_infrastructure/buildsys.py

```diff
@@ -101,8 +105,6 @@ def subtitle(self, msg, **config):
                 tmpl = tmpl[len("{owner}'s "):]
 
             return tmpl.format(inst=inst, **msg['msg'])
-        else:
```

Comment:
Because it spams us with errors when a new message gets introduced instead of just returning an empty string.

I didn't even realize these errors were happening until pingou noticed his inbox getting full.  With this change, the messages would still pass through to IRC, but as obviously new messages that need to be dealt with.